windows/src/desktop/locale/locale-to-android-strings.xsl

```diff
@@ -5,7 +5,9 @@
 	<xsl:template match="String">
     <string>
       <xsl:attribute name="name"><xsl:value-of select="@Id" /></xsl:attribute>
-      <xsl:attribute name="comment"><xsl:value-of select="@Group" />/<xsl:value-of select="@Type" />: <xsl:value-of select="@Description" /></xsl:attribute>
+      <xsl:attribute name="comment"><xsl:value-of select="@Group" />/<xsl:value-of select="@Type" />: <xsl:value-of select="@Description" 
+        /><xsl:if test="@Parameters"> [<xsl:value-of select="@Parameters" />]</xsl:if
+        ><xsl:if test="@Version"> (introduced <xsl:value-of select="@Version" />)</xsl:if></xsl:attribute>
```

Comment:
Do you want to handle the attribute `translatable`?
An example from KMEA strings.xml
```
    <!-- Shared preferences name -->
    <string name="kma_prefs_name" translatable="false">KMAPreferences</string>
```

Reply:
I think all strings listed are translatable 😁  (also, these xsl files will be unnecessary once we complete the transition).